core/src/main/scala/com/rackspace/com/papi/components/checker/step/results/MultiFailResult.scala

```diff
@@ -87,5 +88,5 @@ class MultiFailResult(val fails : Array[Result], stepId : String) extends ErrorR
     }
   }
 
-  override def path : String = startPath+stepIDs.reduceLeft(_+" "+_)+" " + fails.map( _.path ).reduceLeft( _ + " " + _ ) + endPath+"*"+selectId(this)+"*"
+  override def path : String = startPath+super.stepIDs.reduceLeft(_+" "+_)+" " + fails.map( _.path ).reduceLeft( _ + " " + _ ) + endPath+"*"+selectId(this)+"*"
```

Comment:
Sorry, I'm confused here, why isn't calling stepIDs the right thing to do.  Don't we need our steps and the super.steps combined?